port/zos390/omrvmem_support_below_bar_64.s

```diff
@@ -204,30 +204,6 @@ OMRSTORAGE RMODE ANY                                                     000000
 @@CCN@19 ALIAS C'omrfree_memory_below_bar'                               000000
 @@CCN@11 ALIAS C'omrallocate_4K_pages_below_bar'                         000000
 @@CCN@1  ALIAS C'omrallocate_1M_pageable_pages_below_bar'                000000
-************************************************************************
```

Comment:
I believe the duplicate copy was there so that the generated file also contained the copyright

Reply:
That's annoying.  I'll check on that and update here once I've confirmed

Reply:
Compiled on zOS and only see these files:
```
$ cd ~/omr/port
$ find . -name "omrvmem_support_*"                                    
./omrvmem_support_above_bar.o
./omrvmem_support_below_bar_64.o
./zos390/omrvmem_support_below_bar_64.s
./zos390/omrvmem_support_below_bar_31.s
./zos390/omrvmem_support_above_bar.s
```

Is there a specific generated file that needs to be looked at?  I'm not able to find it...